src/views/Substance.vue

```diff
@@ -82,6 +82,9 @@ export default {
   watch: {
     compoundType: function() {
       this.type = this.compoundType;
+    },
+    type: function() {
+      this.$store.dispatch("compound/updateChanged", false);
     }
```

Comment:
This seems like it will cause problems.  `type` changes for reasons that aren't invalidating for a redirect.

For instance:

- Open a ill defined compound
- Edit compound
- Switch type (possibly to another idc type)

Navigating away won't populate the alert.

Reply:
this is a good question but after talking about it I think it will be best to wait for further instruction to figure out exactly what we need before trying to add this type of functionality

Reply:
Functionality looks good.  I can now switch types with new info and it not be unloaded.